knowledge-base/kb-window-close-radwindow-dialog-in-mvvm.md

```diff
@@ -0,0 +1,68 @@
+---
+title: How to close RadWindow dialog in MVVM using custom button
```

Comment:
Same as in the other article. 
- use capital letters for the title
- add description and page_title